packages/inter-protocol/src/proposals/econ-behaviors.js

```diff
@@ -268,16 +284,17 @@ export const setupReserve = async ({
   const STORAGE_PATH = 'reserve';
   trace('setupReserve');
   const poserInvitationP = E(committeeCreator).getPoserInvitation();
-  const [poserInvitation, poserInvitationAmount, feeMintAccess] =
+  const [poserInvitation, poserInvitationAmount, feeMintAccess, ammFacets] =
     await Promise.all([
       poserInvitationP,
       E(E(zoe).getInvitationIssuer()).getAmountOf(poserInvitationP),
       feeMintAccessP,
+      ammFacetsP,
     ]);
 
   const reserveTerms = makeReserveTerms(
     poserInvitationAmount,
-    ammInstanceWithoutReserve,
+    ammFacets.instanceWithoutReserve,
```

Comment:
this is confusing because `instance` isn't a facet. Maybe just call this `ammKit` to be less specific?

Reply:
true. `Kit` is also shorter (both in characters and in syllables) than `Facets`.

If it were a more widely exposed API, I might feel more strongly about it, but in this case, I don't mind much.